nemesyst_core/mongo.py

```diff
@@ -54,8 +54,8 @@ def __init__(self, args=None, logger=None):
             "db_name": "nemesyst",
             "db_collection_name": "test",
             "db_port": "27017",
-            "db_path": self.home + "/db",
-            "db_log_path": self.home + "/db" + "/log",
+            "db_path": "db",
```

Comment:
this now make nemesyst generate mongodb in the local directory, necessary for testing etc purposes without affecting a global position.